sda/internal/database/db_functions.go

```diff
@@ -782,3 +782,21 @@ func (dbs *SDAdb) addKeyHash(keyHash, keyDescription string) error {
 
 	return nil
 }
+
+func (dbs *SDAdb) SetKeyHash(keyHash, fileID string) error {
+	dbs.checkAndReconnectIfNeeded()
+	db := dbs.DB
+
+	query := "UPDATE sda.files SET key_hash = $1 WHERE id = $2;"
+	result, err := db.Exec(query, keyHash, fileID)
+	if err != nil {
+
+		return err
+	}
+	if rowsAffected, _ := result.RowsAffected(); rowsAffected == 0 {
```

Comment:
Why ignore the potentially returned error message?

Reply:
That call will only return an error if the database driver doesn't support returning the number of affected rows in a query and since the SQL driver supports it we can safely ignore it.

Reply:
aha, I see.